### ISSUE TITLE
Let's start a conversation here

### ISSUE THREAD

**Issue by keenanyafiqy** [aieuo/Mineflow#103]
### Question

Let's start an conversation here
Ask anything in here about plugins and others

**Comment by keenanyafiqy** [comment]
so, if we add a new command, it's added. but it isn't listed in command list. does the plugin works like that?

**Comment by aieuo** [comment]
I think that will be listed after restarting the server or rejoining the server.

**Comment by aieuo** [comment]
> Any idea how to make it so you can only execute a recipe 1 time?

One way I think is to set the flag using a global variable and check it at the beginning of the recipe.

**Comment by sweatyrayan** [comment]
how? Idk how to do it

**Comment by aieuo** [comment]
I made a [recipe](https://github.com/aieuo/MineflowExamples/blob/master/Counter.json) to implement it simply.  It works with the latest build.
You can use the recipes, `Callback the other recipe.` action, `If` action, `Exit the recipe` action, and a `Compare the numbers.` condition like this:
![image](https://user-images.githubusercontent.com/35859665/150668428-5aef5945-07e6-4fc5-bcd5-88a566fa44db.png)
![image](https://user-images.githubusercontent.com/35859665/150668456-232f9a42-95c6-4c54-8a41-b4783aa9ed24.png)

The `<args>` of the second image is to identify the count. You can edit it, but you need to specify two values separated by a comma.

**Comment by sweatyrayan** [comment]
Call back the other recipe action is gone? I cant find it anymore, and where is the particles action?

**Comment by aieuo** [comment]
Run `/mineflow permission <your name> 2` from the console and you can see the actions.

**Comment by sweatyrayan** [comment]
I already have my permission to 2

**Comment by aieuo** [comment]
I'm sorry for the late reply.  I tried it on my server and the action is listed below the `Wait`.  
Please open the `plugin_data/Mineflow/player.yml` and make sure your full name is listed like this:
![image](https://user-images.githubusercontent.com/35859665/151645028-b345531d-5810-4e7a-8960-050bada7c293.png)
![image](https://user-images.githubusercontent.com/35859665/151644850-01b047d6-1153-4c46-a8a7-975956c7ceac.png)

**Comment by sweatyrayan** [comment]
No worries
My name wasnt listed in te file, I did it manually and it works now!
I think something in my server is causing it to not work.
Thanks for helping

**Comment by keenanyafiqy** [comment]
@aieuo Do you have any discord server for this plugin? If doesn't, try make a new discord server for this plugin or yours and add it into README for a link discord server

**Comment by keenanyafiqy** [comment]
I may can became a Translator role in your discord server. If you want to

**Comment by keenanyafiqy** [comment]
and last, maybe I can now set-up a PMMP Server 4.0.0 like normally now

**Comment by sweatyrayan** [comment]
I would love a discord server for Mineflow.

**Comment by aieuo** [comment]
> @aieuo Do you have any discord server for this plugin? If doesn't, try make a new discord server for this plugin or yours and add it into README for a link discord server

I don't think it is necessary because we can use the issues and the [discussions](https://github.com/aieuo/Mineflow/discussions) on Github
